docs/index.asciidoc

```diff
@@ -43,14 +44,84 @@ filter plugins.
 
 &nbsp;
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: unstructured geo data added at root level
+** `v1`: uses fields that are compatible with Elastic Common Schema (for example, `[client][geo][country_name]`)
+* Default value depends on which version of Logstash is running:
+** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
+** Otherwise, the default value is `disabled`.
+
+Controls this plugin's compatibility with the {ecs-ref}[Elastic Common Schema (ECS)].
+The value of this setting affects the _default_ value of `type` and `tags`, see <<plugins-{type}s-{plugin}-clones>>
+
 [id="plugins-{type}s-{plugin}-clones"]
 ===== `clones` 
 
   * This is a required setting.
   * Value type is <<array,array>>
   * There is no default value for this setting.
 
-A new clone will be created with the given type for each type in this list.
+ECS Compatibility disabled: a new clone will be created with a `type` of the given type in this list.
+
+ECS Compatibility enabled: a new clone will be created with a `tag` of the given type in this list.
+
+Example:
+[source,ruby]
+    filter {
+      clone {
+        clones => ["sun", "moon"]
+      }
+    }
+
+ECS disabled
+[source,text]
+-----
+{
+      "@version" => "1",
+      "sequence" => 0,
+       "message" => "some messages",
```

Comment:
:+1: for the example maybe a better message than `"some messages"` 
or at least a less confusing singular `"some message"` :wink: